content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -237,7 +237,35 @@ Inter-pod affinity and anti-affinity allow you to constrain which nodes your
 Pods can be scheduled on based on the labels of **Pods** already running on that
 node, instead of the node labels.
 
-Inter-pod affinity and anti-affinity rules take the form "this
+#### Existing Pods' Affinity and Anti-Affinity Considerations
```

Comment:
This is more of a detail of the feature, so it should be after "type of inter-pod affinity and anti-affinity"

Reply:
@alculquicondor Sure, I will update on it .